### DISCUSSION TITLE
FAQs

### DISCUSSION THREAD

**Discussion by seandotau** [subquery/subql#1054] [Frontier TestNet - General]
Hey all, 
We are putting together a list of Frequently Asked Questions at https://www.notion.so/subquery/FAQs-e8b4ad36872e4161a191556ea8a32811#9a73b33598d046b8aa939481600ae560 and would love everyone's input. We'll then review and incorporate it into the FAQ.

**Comment by mzxyz** [comment]
Feel free to leave the questions you have when running as an indexer or consuming the explorer app, we will review all of them and provide detail answers. This would be helpful for the indexers in next season.

**Comment by mzxyz** [comment]
**Example Question:**

As an indexer, how can I get delegation or find someone purchase my plan?

**Reply by mzxyz** [reply]
You can create a new account with MetaMask and get some DEV or SQT tokens in faucet, after that you can use this account as a delegator or consumer to make the action

**Comment by mzxyz** [comment]
**Question:**

As an indexer, what I should do if my indexing project stuck on a specific block height?

**Comment by vostok3** [comment]
What is an acceptable downtime for a node?

**Reply by mzxyz** [reply]
Good question, we do have a check for the node status, the status will be marked as offline if the node is down for a specific time which is config in the contract, we will announce this time duration when season3 start.

**Comment by geramajachkov** [comment]
Should I leave the indexer on until the third season?

**Reply by mzxyz** [reply]
You can safely shutdown the service as for season3 we will do it on Acala, and all data will start from scratch.

**Reply by igorboricev9** [reply]
Can I get detailed information about Acala?

**Reply by mzxyz** [reply]
https://acala.network/acala/learn

**Comment by KorGinKaowNoi** [comment]
this oppotunity is so pure and i can't thank you enough for having me in ss2, can't wait for ss3 <3

**Comment by khristian82dragomirov** [comment]
Thank you for giving me the opportunity to participate in ss2. Looking forward to next season!
